ci/test.yml

```diff
@@ -176,7 +176,7 @@ suite web canary fws manual:
     matrix:
       - TEST_FILE: ["spawn-tor", "spawn-bridge"]
         CONTAINERS: "trezor-user-env-unix"
-      - TEST_FILE: ["electrum"]
+      - TEST_FILE: ["electrum", "bug-report"]
```

Comment:
does bug-report test need electrum regtest container?

Reply:
if not, it should be included on line 178. this is the mechanism used not to run unneeded containers

Reply:
otherwise good job, feel free to resolve my review in case I don't react :P